tests/integration/FtpClientTest.php

```diff
@@ -565,6 +568,8 @@ public function testFind() : void
 
     public function testFindRecursive() : void
     {
+        $this->markTestIncomplete('The find method with recursive approach seems to be problematic.');
```

Comment:
The `find` method with recursive seems to be problematic because it will not find directory recursively.

For example, if we want find the text files that are located on `./test_dir/test_dir`, the above method will list the `./test_dir` and find the text file on `./test_dir`.

Reply:
Ehmm, but I see the tests are already passed successfully according to your [GitHub actions log](https://github.com/open-source-contributions/php-ftp-client/actions/runs/1850399314).

Reply:
I've marked this test and it will be ignored during the test. And the test will be passed successfully.

If removing this line and run that, it will be failed:

```Bash
lee@lee-VirtualBox:~/php-ftp-client$ vendor/bin/phpunit --filter testFindRecursive tests/integration/FtpClientTest.php
PHPUnit 9.5.13 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.4.28
Configuration: /home/lee/php-ftp-client/phpunit.xml

F                                                                   1 / 1 (100%)

Time: 00:00.150, Memory: 6.00 MB

There was 1 failure:

1) Lazzard\FtpClient\Tests\Integration\FtpClientTest::testFindRecursive
Failed asserting that an array is not empty.

/home/lee/php-ftp-client/tests/integration/FtpClientTest.php:580
phpvfscomposer:///home/lee/php-ftp-client/vendor/phpunit/phpunit/phpunit:97
```

If we change into the `$deepDir` on the line `580` (`$this->assertNotEmpty($client->find('/.*\.txt$/i', $deepDir, true));`), it will be passed:

```Bash
lee@lee-VirtualBox:~/php-ftp-client$ vendor/bin/phpunit --filter testFindRecursive tests/integration/FtpClientTest.php
PHPUnit 9.5.13 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.4.28
Configuration: /home/lee/php-ftp-client/phpunit.xml

.                                                                   1 / 1 (100%)

Time: 00:00.074, Memory: 6.00 MB

OK (1 test, 1 assertion)
```

Reply:
According to above experiment, we can see that the `find` method with recursive mode is problematic.

Reply:
This is strange actually, I've run this test with a real FTP server before and everything was working just fine, however ignoring this test is fun for now, I'll debug this later.